src/components/navigation/desktopnavigation/DesktopNavigation.tsx

```diff
@@ -5,72 +5,92 @@ import RoutingPath from '../../../routes/RoutingPath'
 import { useContext } from 'react'
 import { UserContext } from '../../../shared/providers/UserProvider'
 import { Profile } from '../profile/Profile'
+import { DESKTOP_NAV_HEIGHT } from '../../../shared/styles/constants'
 
 export const DesktopNavigation = () => {
 	const history = useHistory()
-	const [authenticatedUser,] = useContext(UserContext)
+	const [authenticatedUser] = useContext(UserContext)
 
 	const displayAuthentication = () => {
-		return authenticatedUser.authenticated
-			? <Profile />
-			: <Button onClick={() => history.push(RoutingPath.signInView)}>Logga in</Button>
+		return authenticatedUser.authenticated ? (
+			<Profile />
+		) : (
+			<Button onClick={() => history.push(RoutingPath.signInView)}>
+        Logga in
+			</Button>
+		)
 	}
```

Comment:
Prettier in action?
Formatting looks uglaay ;<

Reply:
@ArastoSahbaei  I think Eslint did that